tests/test_validate.py

```diff
@@ -76,6 +80,14 @@ def test_validate_environment_marker():
     assert len(res) == 1
     assert res[0].startswith("Invalid expression")
 
+    res = vem("""()))))()extra == "test"(((((((""")  # No chained comparisons
+    assert len(res) == 1
+    assert res[0] == 'Invalid expression. incorrect parentheses'
+
+    res = vem("""extra == "test" and or (os_name == "nt" or python_version == "2.7")""")  # No chained comparisons
+    assert len(res) == 1
+    assert res[0] == "Invalid expression \"and or\""
+
```

Comment:
Can we add  a test for an empty pair of parentheses, like `extra == "test" and ()`?